pkgs/tools/misc/colorless/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+, coreutils
+, dash
+, gnused
+, less
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "colorless";
+  version = "109";
+
+  src = fetchurl {
+    url = "http://software.kimmo.suominen.com/${pname}-${version}.tar.gz";
+    sha256 = "039a140fa11cf153cc4d03e4f753b7ff142cab88ff116b7600ccf9edee81927c";
+  };
+
+  makeFlags = [
+    "TOOLPATH=${coreutils}/bin:${gnused}/bin:${less}/bin"
+    "PREFIX=${placeholder "out"}"
+    "SHELL=${dash}/bin/dash"
+  ];
+
+  preInstall = ''
+    mkdir -p $out/bin $out/share/man/man1
+  '';
+
+  postInstall = ''
+    mkdir -p $out/share/doc/colorless
+    install -Dm644 -t $out/share/doc/colorless LICENSE README.md
```

Comment:
`install` should automatically create the required directories for you.

Reply:
Removed `mkdir` -- thank you.